src/freenet/node/LocationManager.java

```diff
@@ -193,11 +195,12 @@ public void run() {
 
             @Override
             public void run() {
-                node.ticker.queueTimedJob(this, PITCH_BLACK_MITIGATION_FREQUENCY_ONE_DAY);
+                LocalDateTime now = LocalDateTime.now(clockForTesting);
```

Comment:
Is this clock not just for running (unit?) tests? Why use it at runtime?

Reply:
It’s actually a system-clock running at UTC that can be replaced for testing, so the naming was not that great. I renamed it to systemClockUTC now.

Thank you!